cmd/podman/pod_rm.go

```diff
@@ -22,12 +22,13 @@ var (
 		},
 		LatestFlag,
 	}
-	podRmDescription = "Remove one or more pods"
-	podRmCommand     = cli.Command{
-		Name: "rm",
-		Usage: fmt.Sprintf(`podman rm will remove one or more pods from the host. The pod name or ID can be used.
-                            A pod with containers will not be removed without --force.
-                            If --force is specified, all containers will be stopped, then removed.`),
+	podRmDescription = fmt.Sprintf(`
+podman rm will remove one or more pods from the host. The pod name or ID can
+be used.  A pod with containers will not be removed without --force.            If --force is specified, all containers will be stopped, then removed.
```

Comment:
does the tab/spaces before the 'If' look OK?

Reply:
Nope, supposed to be a new line.